packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/input/InputEditorCellSpec.js

```diff
@@ -90,57 +90,14 @@ describe('decision-table-head/editor - input', function() {
   }));
 
 
-  describe('should transform to script', function() {
```

Comment:
Removing the script feature isn't part of this PR, right? Maybe it makes sense to split the commit in removing the test case and the original improvement aka setting the default expression language.

Reply:
I will do that.

Reply:
Done via https://github.com/bpmn-io/dmn-js/pull/537/commits/7380a36254eb3be8bac14935a293f0a1b84945c8